browser/brave_profile_prefs.cc

```diff
@@ -128,6 +128,10 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(kNewTabPageShowClock, true);
   registry->RegisterBooleanPref(kNewTabPageShowTopSites, true);
   registry->RegisterBooleanPref(kNewTabPageShowStats, true);
+
+  // Brave Wallet
+  registry->RegisterStringPref(kBraveWalletAES256GCMSivNonce, "");
```

Comment:
per comment in https://github.com/brave/internal/issues/633#issuecomment-520635998, it is actually ok to just use a nonce of all 0's (so no need to store anything). but not critical to change this.

Reply:
I think I'll just leave it like this since it's already done and tested.